runtime/Stdlib_BigInt.res

```diff
@@ -35,7 +35,13 @@ switch BigInt.fromStringExn("a") {
 */
 external fromStringExn: string => bigint = "BigInt"
 @val external fromInt: int => bigint = "BigInt"
-@val external fromFloat: float => bigint = "BigInt"
+@val external _fromFloat: float => bigint = "BigInt"
```

Comment:
As we don't have a .resi, maybe it's better to call this `fromFloat`, too, and shadow it below, so that `_fromFloat` is not exported?

Reply:
yup that's better, updated.